code/modules/roguetown/roguecrafting/items.dm

```diff
@@ -97,6 +97,24 @@
 	verbage_simple = "carve"
 	verbage = "carves"
 
+/datum/crafting_recipe/roguetown/recurvebowpartial
+	name = "unstrung recurve bow"
+	result = /obj/item/grown/log/tree/recurvebowpartial
+	reqs = list(/obj/item/grown/log/tree/small = 1,/obj/item/natural/hide/cured = 1)
+	tools = /obj/item/rogueweapon/huntingknife
+	verbage_simple = "carve"
+	verbage = "carves"
+	craftdiff = 3
+
+/datum/crafting_recipe/roguetown/longbowpartial
+	name = "unstrung longbow"
+	result = /obj/item/grown/log/tree/longbowpartial
+	reqs = list(/obj/item/grown/log/tree = 1)
```

Comment:
This is unobtainable, should be log/tree/large

Reply:
fixed